pkg/kubectl/bind/plugin/bind.go

```diff
@@ -55,8 +55,9 @@ type BindOptions struct {
 	*base.Options
 	Logs *logs.Options
 
-	Print  *genericclioptions.PrintFlags
-	DryRun bool
+	PrintFlags *genericclioptions.PrintFlags
```

Comment:
it was intentionally called Print, not PrintFlags. See how all other options here are not FooOptions, but Foo.